mlflow/utils/rest_utils.py

```diff
@@ -12,6 +12,7 @@
 from mlflow.utils.string_utils import strip_suffix
 from mlflow.exceptions import MlflowException, RestException
 
+_PATH_PREFIX = "/api/2.0"
```

Comment:
When I tried to import `_PATH_PREFIX` from  `databricks_artifact_repo.py`, I got an error due to a circular import.

https://github.com/mlflow/mlflow/blob/d743a40426d5dedbde395a4e6bbdeebadbccd4dc/mlflow/store/artifact/databricks_artifact_repo.py#L36

Reply:
I think we should define `_PATH_PREFIX` somewhere and reuse it rather than defining it in each file. The following files contains `_PATH_PREFIX`.

- `mlflow/store/artifact/databricks_artifact_repo.py`
- `mlflow/store/model_registry/rest_store.py`
- `mlflow/store/tracking/rest_store.py`

Reply:
SGTM to reuse the constant, can we name it something more descriptive like `_REST_API_PATH_PREFIX`?